src/main/java/Main.java

```diff
@@ -1,2 +1,24 @@
+import java.sql.SQLException;
+
+import org.apache.arrow.vector.FieldVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+
 public class Main {
+  public static void main(String[] args) throws Exception {
+    String userName = args[0];
+    String password = args[1];
+    String urlPort = args[2];
+
+    PostgresDriverWrapper driver = new PostgresDriverWrapper(urlPort, userName, password);
+    driver.connect();
+    VectorSchemaRoot vectorSchemaRoot = driver.executeQuery("select * from personaldata");
+    driver.close();
```

Comment:
If you are implementing AutoClosable, close the used resources using try-with-resources

Reply:
done